src/sdx/pce/heuristic/heur.py

```diff
@@ -0,0 +1,274 @@
+import argparse
+import json
+
+# importing the module
+from datetime import datetime
+
+import numpy as np
+import prtpy
+
+from sdx.pce.load_balancing.te_solver import TESolver
+from sdx.pce.models import ConnectionRequest, TrafficMatrix
+from sdx.pce.utils.constants import Constants
+from sdx.pce.utils.random_connection_generator import RandomConnectionGenerator
+from sdx.pce.utils.random_topology_generator import RandomTopologyGenerator
+
+
+def random_graph(n, p, m):
```

Comment:
Can we have a docstring to describe what this function does, including parameters?  Also would be nice to use more descriptive parameter names.